### ISSUE TITLE
Computational Requirements

### ISSUE THREAD

**Issue by asif-hanif** [StanfordMIMI/DDM2#3]
Hi @tiangexiang 
Thanks for great work.

Could you provide information about computational requirements of the model training from Stage-I through Stage-III i.e. number of GPUs used and approximate total training time?

**Comment by tiangexiang** [comment]
Hi, thanks for your interest in our work! Most of our experiments (from Stage-I through Stage-III) are able to run on GPUs with 11GB memory. The training of Stage-I may take up to 5~6 hours, while the training of Stage-III may take up to a day, depending on the computing resources you have. Thanks!

**Comment by asif-hanif** [comment]
Thanks for sharing the information. I will close the issue now.

**Comment by tiangexiang** [comment]
Hi, I have looked into this further and double-checked the training log, the running time stats are as follows:

Stage I: ~52 s for 1,000 steps, ~90 mins for all 100,000 steps training. 
Stage II: ~1-2 hours, depending on the size of the 4D MRI.
Stage III: ~140 s for 1,000 steps, ~23-24 hours for all 100,000 steps training.

We ran all of the experiments on a single RTX 2080-Ti GPU with 11 GB memory.
